src/goto-harness/recursive_initialization.cpp

```diff
@@ -426,49 +332,421 @@ std::string recursive_initialization_configt::to_string() const
   return out.str();
 }
 
-recursive_initializationt::array_convertert
-recursive_initializationt::default_array_member_initialization()
+irep_idt recursive_initializationt::get_fresh_global_name(
+  const std::string &symbol_name,
+  const exprt &initial_value) const
+{
+  symbolt &fresh_symbol = get_fresh_aux_symbol(
+    signed_int_type(),
+    "__goto_harness",
```

Comment:
Probably makes more sense to use `CPROVER_PREFIX`, same in following

Reply:
Done.